voronoi_game.py

```diff
@@ -103,29 +107,15 @@ def __get_player_move(self):
     data = client_response.split()
     return int(data[0]), int(data[1])
 
+  def compute_pull(self, row, col):
+    return np.reciprocal(np.sqrt(np.square(self.row_numbers - row) + np.square(self.col_numbers - col) + 0.1e-30))
```

Comment:
Is this calculating the distance from a point to the corner of the map? Why would we want that? Also why the `0.1e-30`?

Reply:
this is calculating the additional pull for every point in the grid.

Also, 0.1e-30 takes in account that there will be one point where distance = 0. So it makes it a very very small number and it has no other impact on points where distance is considerably larger than 0.1e-30 already.

Reply:
So, even if distance = 0, the reciprocal won't throw a division by zero error but will generate a very very big number.